samples/knowledge_base_management.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Dialogflow API Python sample showing how to manage Knowledge bases.
+
+Examples:
+  python knowledge_base_management.py -h
+  python knowledge_base_management.py --project-id PROJECT_ID \
+  list
+  python knowledge_base_management.py --project-id PROJECT_ID \
+  create --display-name DISPLAY_NAME
+  python knowledge_base_management.py --project-id PROJECT_ID \
+  get --knowledge-base-id knowledge_base_id
+  python knowledge_base_management.py --project-id PROJECT_ID \
+  delete --knowledge-base-id knowledge_base_id
+"""
+
+import argparse
+
+
+# [START dialogflow_list_knowledge_base]
+def list_knowledge_bases(project_id):
+    """Lists the Knowledge bases belonging to a project.
+
+    Args:
+        project_id: The GCP project linked with the agent."""
+    import dialogflow_v2beta1 as dialogflow
+    client = dialogflow.KnowledgeBasesClient()
+    agent_path = client.project_path(project_id)
```

Comment:
is there also a `client.agent_path` helper method?  i think currently `project_path` and `agent_path` are still the same thing, but here which one are we using?

Reply:
Updated to `project_path`

Reply:
project_path used